sig-cli/CONTRIBUTING.md

```diff
@@ -288,6 +267,9 @@ Implementation PRs should
 - mention `@kubernetes/sig-cli-pr-reviews`,
```

Comment:
I don't think any of these "mention `@kubernetes/sig-cli-pr-reviews`" and others like it are still relevant.

It looks like these might refer to various google groups that aren't really used anymore:
https://groups.google.com/g/kubernetes-sig-cli-pr-reviews